README.md

```diff
@@ -75,16 +76,6 @@ Other possible values are `1` (`MIXED_CONTENT_NEVER_ALLOW`) and `2` (`MIXED_CONT
 
 Preferences only available for iOS platform
 
-#### UseScheme
-
-`<preference name="UseScheme" value="true" />`
-
-Default value is `false`.
-
-On iOS 11 and newer it will use a `WKURLSchemeHandler` that loads the app from `ionic://` scheme instead of using the local web server and `https://` scheme.
-
-On iOS 10 and older will continue using the local web server even if the preference is set to `true`.
-
 #### HostName
```

Comment:
Hostname is one word, so the setting should be `Hostname`. However, if it's Android only perhaps it should be `AndroidHostname`